packages/aws-rfdk/lib/core/lib/cloudwatch-agent.ts

```diff
@@ -114,18 +133,32 @@ export class CloudWatchAgent extends Construct {
    * This is done by adding UserData commands to the target host.
    *
    * @param host The host to configure the CloudWatch agent on
+   * @param shouldInstallAgent Attempts to install the CloudWatch agent on the instance if set to true.
    * @param skipValidation Skips the validation of the CloudWatch agent installer if set to true.
    */
-  private configure(host: IScriptHost, skipValidation: boolean) {
+  private configure(
+    host: IScriptHost,
+    shouldInstallAgent: boolean,
+    skipValidation: boolean,
+  ) {
     // Grant access to the required CloudWatch Agent installer files
     const cloudWatchAgentBucket = Bucket.fromBucketArn(this, 'CloudWatchAgentBucket', 'arn:aws:s3:::amazoncloudwatch-agent');
```

Comment:
Just noticed that we missed updating this. The bucket ARN changes based on the region now, no?

Reply:
Yes good catch, this was definitely a miss in my testing. When running the script through the RFDK example app, I was using the Thinkbox Deadline Linux and Windows base AMI's, which both have the CloudWatch agent pre-installed.

Since I don't have an AMI with Deadline installed but the CloudWatch agent not installed, I commented out the check for an existing agent in both configuration scripts and then turned off the worker fleet's health check so I could deploy the fleets and confirm the failure by logging into the hosts through the Systems Manager and checking the logs. After that, I updated this code to grant permissions for the regional bucket and redeployed to confirm that it was working.